dist/info/3dengine_libretro.info

```diff
@@ -9,3 +9,4 @@ license = "GPLv3"
 permissions = ""
 display_version = "v1"
 supports_no_game = "false"
+notes = "(!) 3DEngine does not use any BIOS files."
```

Comment:
I agree with RobLoach it's redundant. People are less likely to read when there is more text.